CRITICAL - 2022-04-05 07:56:53 --> Call to undefined function CodeIgniter\locale_set_default()
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\bootstrap.php(146): CodeIgniter\CodeIgniter->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\spark(45): require('C:\\xampp\\htdocs...')
#2 {main}
ERROR - 2022-04-05 00:57:33 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-04-05 00:57:33 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Semester 4\ci\app\Models\PortoModel.php(15): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Semester 4\ci\app\Controllers\Home.php(55): App\Models\PortoModel->getEdu()
#4 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Home->index()
#5 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Semester 4\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2022-04-05 01:24:58 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-04-05 01:24:58 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Semester 4\ci\app\Models\PortoModel.php(15): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Semester 4\ci\app\Controllers\Home.php(55): App\Models\PortoModel->getEdu()
#4 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Home->index()
#5 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Semester 4\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2022-04-05 04:37:44 --> Error connecting to the database: No connection could be made because the target machine actively refused it
CRITICAL - 2022-04-05 04:37:44 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Semester 4\ci\app\Models\PortoModel.php(15): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Semester 4\ci\app\Controllers\Home.php(55): App\Models\PortoModel->getEdu()
#4 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Home->index()
#5 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Semester 4\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2022-04-05 04:37:50 --> Error connecting to the database: No connection could be made because the target machine actively refused it
CRITICAL - 2022-04-05 04:37:50 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Semester 4\ci\app\Models\PortoModel.php(15): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Semester 4\ci\app\Controllers\Home.php(55): App\Models\PortoModel->getEdu()
#4 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Home->index()
#5 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Semester 4\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2022-04-05 04:37:54 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-04-05 04:37:54 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Semester 4\ci\app\Models\PortoModel.php(15): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Semester 4\ci\app\Controllers\Home.php(55): App\Models\PortoModel->getEdu()
#4 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Home->index()
#5 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Semester 4\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2022-04-05 04:37:56 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-04-05 04:37:56 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Semester 4\ci\app\Models\PortoModel.php(15): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Semester 4\ci\app\Controllers\Home.php(55): App\Models\PortoModel->getEdu()
#4 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Home->index()
#5 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Semester 4\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2022-04-05 04:37:56 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-04-05 04:37:56 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Semester 4\ci\app\Models\PortoModel.php(15): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Semester 4\ci\app\Controllers\Home.php(55): App\Models\PortoModel->getEdu()
#4 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Home->index()
#5 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Semester 4\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2022-04-05 04:37:57 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-04-05 04:37:57 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Semester 4\ci\app\Models\PortoModel.php(15): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Semester 4\ci\app\Controllers\Home.php(55): App\Models\PortoModel->getEdu()
#4 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Home->index()
#5 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Semester 4\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2022-04-05 04:37:57 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-04-05 04:37:57 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Semester 4\ci\app\Models\PortoModel.php(15): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Semester 4\ci\app\Controllers\Home.php(55): App\Models\PortoModel->getEdu()
#4 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Home->index()
#5 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Semester 4\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2022-04-05 04:37:57 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-04-05 04:37:57 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Semester 4\ci\app\Models\PortoModel.php(15): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Semester 4\ci\app\Controllers\Home.php(55): App\Models\PortoModel->getEdu()
#4 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Home->index()
#5 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Semester 4\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2022-04-05 04:37:58 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-04-05 04:37:58 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Semester 4\ci\app\Models\PortoModel.php(15): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Semester 4\ci\app\Controllers\Home.php(55): App\Models\PortoModel->getEdu()
#4 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Home->index()
#5 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Semester 4\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2022-04-05 04:37:59 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-04-05 04:37:59 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Semester 4\ci\app\Models\PortoModel.php(15): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Semester 4\ci\app\Controllers\Home.php(55): App\Models\PortoModel->getEdu()
#4 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Home->index()
#5 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Semester 4\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2022-04-05 04:38:00 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-04-05 04:38:00 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Semester 4\ci\app\Models\PortoModel.php(15): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Semester 4\ci\app\Controllers\Home.php(55): App\Models\PortoModel->getEdu()
#4 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Home->index()
#5 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Semester 4\ci\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\Semester 4\ci\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
